src/main/java/org/mule/extension/http/api/HttpResponseAttributes.java

```diff
@@ -40,4 +40,20 @@ public String getReasonPhrase() {
     return reasonPhrase;
   }
 
+  public String toString() {
+    StringBuilder builder = new StringBuilder();
+    builder.append("The response has these values\n");
+    builder.append("Status Code: ").append(statusCode);
+    builder.append("\n");
```

Comment:
You could do this in one line. Applies to all these `builder.append("\n")` and the other classes.